net-misc/wget2/wget2-2.1.0.ebuild

```diff
@@ -0,0 +1,96 @@
+# Copyright 1999-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+DESCRIPTION="GNU Wget2 is a file and recursive website downloader"
+HOMEPAGE="https://gitlab.com/gnuwget/wget2"
+SRC_URI="mirror://gnu/wget/${P}.tar.gz"
+
+# LGPL for libwget
+LICENSE="GPL-3+ LGPL-3+"
+SLOT="0/0" # subslot = libwget.so version
+KEYWORDS="~amd64 ~arm64 ~x86"
+IUSE="brotli bzip2 doc +gnutls gpgme +http2 idn lzip lzma openssl pcre psl +ssl test xattr zlib"
+
+RDEPEND="
+	brotli? ( app-arch/brotli )
+	bzip2? ( app-arch/bzip2 )
+	!gnutls? ( dev-libs/libgcrypt:= )
+	ssl? (
+		gnutls? ( net-libs/gnutls:= )
+		!gnutls? (
+			dev-libs/openssl:0=
+		)
+	)
+	gpgme? (
+		app-crypt/gpgme:=
+		dev-libs/libassuan
+		dev-libs/libgpg-error
+	)
+	http2? ( net-libs/nghttp2 )
+	idn? ( net-dns/libidn2:= )
+	lzip? ( app-arch/lzlib )
+	lzma? ( app-arch/xz-utils )
+	pcre? ( dev-libs/libpcre2 )
+	psl? ( net-libs/libpsl )
+	xattr? ( sys-apps/attr )
+	zlib? ( sys-libs/zlib )
+"
+DEPEND="${RDEPEND}"
+BDEPEND="
+	virtual/pkgconfig
+	doc? ( app-doc/doxygen[dot] )
+"
+
+RESTRICT="!test? ( test )"
+
+PATCHES=(
+	#"${FILESDIR}"/${PN}-fix-build-issues-with-clang-16.patch
+	#"${FILESDIR}"/${P}-stdint.patch
+)
```

Comment:
I will clear this directly when committing but for next time, usually you can clear out directly the line (or full PATCHES section), especially for these that have been applied upstream

Reply:
Noted for future reference. Thanks!